### PR TITLE
[TECH] Add React Dev tools debug script

### PR DESCRIPTION
**Pull request by flavioislima**
Install the proper extension and loads it when using `yarn debug:react`

image

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2092,13 +2097,25 @@ ajv@^8.0.0, ajv@^8.6.3:
...
   resolved "https://registry.yarnpkg.com/ansi-escapes/-/ansi-escapes-4.3.2.tgz#6b2291d1db7d98b6521d5f1efa42d0f3a9feb65e"
   integrity sha512-gKXj5ALrKWQLsYG9jlTRmR/xKluxHV+Z9QEwNIgCfM1/uwPMCuzVVnh5mwTd+OuBZcwSIMbqssNWRm1lE51QaQ==
   dependencies:
     type-fest "^0.21.3"
 
+ansi-regex@^3.0.0:
+  version "3.0.1"
+  resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-3.0.1.tgz#123d6479e92ad45ad897d4054e3c7ca7db4944e1"
+  integrity sha512-+O9Jct8wf++lXxxFc4hc8LsjaSq0HFzzL7cVsw8pRDIPdjKD2mT4ytDZlLuSBZ4cLKZFXIrMGO7DbQCtMJJMKw==
+
```
Comment (arielj) [inline yarn.lock:2118]: not sure what happened with this file but there are multiple dependencies duplicated now, ansi-regex 3.0.0 and 5.0.1, camelcase 4.0.0 and 5.0.1, and others

not sure if that is correct or a conflict/merge issue

Comment (flavioislima) [reply yarn.lock:2118]: I have no idea. I just installed react-devtool with yarn.
Maybe we should delete yarn.lock and generate a new one?

Comment (CommandMC) [reply yarn.lock:2118]: These are dependencies of the `react-devtools` package. Other than there are quite a few of them, this is nothing abnormal (it seems that they're depending on an older version of Electron, which in turn pulls in quite a few other older packages)

**Comment by arielj** [comment]
merged https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3765 that replaces this PR

**Pull request closed without merging**